content/blog/automating-operations-on-dscc-using-ansible-playbooks.md

```diff
@@ -0,0 +1,437 @@
+---
+title: Automating operations on DSCC using Ansible Playbooks
```

Comment:
DSCC --> Data Services Cloud Console 

Please use full naming rules for DSCC - do not use the acronym DSCC throughout the post and for the title.